### ISSUE TITLE
Failed Compilation

### ISSUE THREAD

**Issue by Lecale** [GiovineItalia/Gadfly.jl#1188]
Fresh install for Julia 1, Win10, if you need more info just ask

```
julia> using Gadfly
[ Info: Precompiling Gadfly [c91e804a-d5a3-530f-b6f0-dfbca275c004]
ERROR: LoadError: syntax: try without catch or finally
Stacktrace:
 [1] include at .\boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at .\loading.jl:1038
 [3] include(::Module, ::String) at .\sysimg.jl:29
 [4] top-level scope at none:2
 [5] eval at .\boot.jl:319 [inlined]
 [6] eval(::Expr) at .\client.jl:389
 [7] top-level scope at .\none:3
in expression starting at C:\Users\sivad\.julia\packages\Gadfly\ojGNo\src\Gadfly.jl:971
ERROR: Failed to precompile Gadfly [c91e804a-d5a3-530f-b6f0-dfbca275c004] to C:\Users\sivad\.julia\compiled\v1.0\Gadfly\DvECm.ji.

Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1184
 [4] macro expansion at .\logging.jl:311 [inlined]
 [5] _require(::Base.PkgId) at .\loading.jl:941
 [6] require(::Base.PkgId) at .\loading.jl:852
 [7] macro expansion at .\logging.jl:311 [inlined]
 [8] require(::Module, ::Symbol) at .\loading.jl:834
```

**Comment by tlnagy** [comment]
Gadfly doesn't yet support Julia 1.0 yet (see progress in https://github.com/GiovineItalia/Gadfly.jl/issues/1158) so we recommend using Julia v0.6.4 for now. We expect to support 1.0 in the next week or two

**Comment by Lecale** [comment]
(v0.7) pkg> add Gadfly
   Cloning default registries into C:\Users\u0159027\.julia\registries
   Cloning registry General from "https://github.com/JuliaRegistries/General.git
"
  Updating registry at `C:\Users\u0159027\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed DiffRules ────────────── v0.0.7
 Installed ShowItLikeYouBuildIt ─── v0.2.0
 Installed PDMats ───────────────── v0.9.5
 Installed OrderedCollections ───── v0.1.0
 Installed PositiveFactorizations ─ v0.2.1
 Installed DataArrays ───────────── v0.7.0
 Installed StatsBase ────────────── v0.25.0
 Installed Gadfly ───────────────── v0.8.0
 Installed VersionParsing ───────── v1.1.2
 Installed StatsFuns ────────────── v0.7.0
 Installed URIParser ────────────── v0.4.0
 Installed DiffEqDiffTools ──────── v0.6.0
 Installed SpecialFunctions ─────── v0.7.0
 Installed DocStringExtensions ──── v0.5.0
 Installed DataStructures ───────── v0.12.0
 Installed Rmath ────────────────── v0.5.0
 Installed DataStreams ──────────── v0.4.0
 Installed Measures ─────────────── v0.3.0
 Installed SortingAlgorithms ────── v0.3.1
 Installed Compat ───────────────── v1.1.0
 Installed TranscodingStreams ───── v0.8.1
 Installed Distributions ────────── v0.16.4
 Installed Optim ────────────────── v0.16.0
 Installed Calculus ─────────────── v0.4.1
 Installed IterTools ────────────── v1.0.0
 Installed Juno ─────────────────── v0.5.3
 Installed CoupledFields ────────── v0.1.0
 Installed Conda ────────────────── v1.0.1
 Installed JSON ─────────────────── v0.19.0
 Installed StaticArrays ─────────── v0.8.3
 Installed CodecZlib ────────────── v0.5.0
 Installed Reexport ─────────────── v0.2.0
 Installed Ratios ───────────────── v0.3.0
 Installed Arpack ───────────────── v0.2.3
 Installed CommonSubexpressions ─── v0.2.0
 Installed AxisAlgorithms ───────── v0.3.0
 Installed ColorTypes ───────────── v0.7.4
 Installed DataFrames ───────────── v0.13.1
 Installed BinaryProvider ───────── v0.4.2
 Installed CategoricalArrays ────── v0.3.13
ERROR: Error when installing packages:
IOError: unlink: operation not permitted (EPERM)
Stacktrace:
 [1] uv_error at .\libuv.jl:85 [inlined]
 [2] unlink(::String) at .\file.jl:743
 [3] #rm#9(::Bool, ::Bool, ::Function, ::String) at .\file.jl:253
 [4] #rm at .\none:0 [inlined]
 [5] macro expansion at .\logging.jl:314 [inlined]
 [6] install_archive(::Array{String,1}, ::Base.SHA1, ::String) at C:\cygwin\home
\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\P
kg\src\Operations.jl:452
 [7] macro expansion at C:\cygwin\home\Administrator\buildbot\worker\package_win
64\build\usr\share\julia\stdlib\v0.7\Pkg\src\Operations.jl:570 [inlined]
 [8] (::getfield(Pkg.Operations, Symbol("##21#24")){Pkg.Types.Context,Dict{Base.
UUID,Base.SHA1},Dict{Base.UUID,Array{String,1}}})() at .\task.jl:262

(v0.7) pkg> ]
ERROR: expected command. instead got []]

shell>

**Comment by tlnagy** [comment]
Closed in favor of #1185

**Comment by tlnagy** [comment]
@Lecale We don't yet support 1.0. We're really close. See #1189 for details.
